app/scripts/modules/core/delivery/executionGroup/execution/execution.directive.html

```diff
@@ -30,6 +30,7 @@
     </div>
     <div class="execution-summary">
       Status: <span class="status execution-status execution-status-{{::vm.execution.status.toLowerCase()}}">{{::vm.execution.status}}</span>
+      <span ng-if="::vm.execution.cancellationReason" class="glyphicon glyphicon-info-sign" uib-tooltip="{{ ::vm.execution.cancellationReason }}"></span>
```

Comment:
I think this would stand out a little more if it came at the end, i.e. after the cancellation timestamp. But I'm fine with it in its current location if you feel strongly about it.